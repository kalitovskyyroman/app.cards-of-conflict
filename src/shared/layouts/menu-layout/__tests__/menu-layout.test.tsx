import '@testing-library/jest-dom';
import { render, screen } from '@testing-library/react';
import MenuLayout from '../menu-layout';

describe('MenuLayout component', () => {
    it('Render component', () => {
        const { container } = render(<MenuLayout>text</MenuLayout>);
        expect(container).toMatchSnapshot();
    });

    it('Render with text', () => {
        const mockText = 'mock text';
        const testId = 'test-id';

        render(
            <MenuLayout>
                <div data-testid={testId}>{mockText}</div>
            </MenuLayout>
        );

        expect(screen.getByTestId(testId)).toHaveTextContent(mockText);
    });
});
