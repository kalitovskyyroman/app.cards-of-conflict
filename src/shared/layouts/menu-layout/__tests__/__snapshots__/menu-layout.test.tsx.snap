// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenuLayout component Render component 1`] = `
<div>
  <div
    class="layout"
  >
    text
  </div>
</div>
`;
